.github/workflows/patch.yaml

```diff
@@ -0,0 +1,132 @@
+name: Patch Images
+
+on:
+  schedule:
+    - cron: '0 2 */15 * *'  # Every 15 days at 2 a.m.
+#   push:
+#     branches:
+#       - main
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+        mymatrix: ${{ steps.dataStep.outputs.myoutput }}
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+
+      # Retrieve image list via python (you can choose to include the last 3 tags by passing the '--include-last-3-tags' argument to the script)
+      - name: Export Image List With Python
+        id: dataStep
+        run: |
+          cd utilities
+          TARGETS=$(python3 image_list_json.py)
+          echo $TARGETS
+          echo "myoutput=$(jq -cn --argjson environments "$TARGETS" '{target: $environments}')" >> $GITHUB_OUTPUT
+
+  patch:
+    needs: setup
+    runs-on: ubuntu-latest
+    strategy:
+        fail-fast: false
+        matrix: ${{fromJson(needs.setup.outputs.mymatrix)}}
+    steps:
+    - name: Print image name
+      run: echo ${{matrix.target}}
+
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@v3.1.1
+      with:
+        cosign-release: 'v2.2.3'
+
+    - name: Generate Trivy Report
+      uses: aquasecurity/trivy-action@69cbbc0cbbf6a2b0bab8dcf0e9f2d7ead08e87e4
+      with:
+        scan-type: 'image'
+        format: 'json'
+        output: 'report.json'
+        ignore-unfixed: true
+        vuln-type: 'os'
```

Comment:
You are most probably aware of this, but I'd say it just in case:

`vuln-type: os` means that `trivy` will look for software installed via the OS _package manager_ and not packages installed with other tools like pip, npm, brew, etc.

This does **not** guarantee that the application you want from that image won't be updated too, it could have been installed via a "os" package (.deb, .rpm, etc.). A side effect of this is that dependencies that may have vulnerabilities installed via other package managers (like pip) won't be detected.

Reply:
Yes, we are aware of this fact, but at the moment, *Copacetic* is only capable of automatically fixing dependencies installed via packet managers: *apt*, *yum*, *apk*.

Reply:
Please add this limitation to the PR description so it is clear in the future and this does not get lost in a GitHub comment.